### DISCUSSION TITLE
(Typhoon) and Tailwindcss 3

### DISCUSSION THREAD

**Discussion by ghost** [getgrav/grav-premium-issues#349] [Q&A]
i’m very interested in creating a “mega menu” like explained in : [Tailwind CSS Mega Menu - Flowbite](https://flowbite.com/docs/components/mega-menu/#full-width-with-image)
(Paragraph “Full width with image”)

i hope to create a menu like it : https://www.jpl.nasa.gov/

My Question : does Typhoon have this capability ?

Many thanks for your help.
Stephan

**Comment by rhukster** [comment]
Typhoon does not have a megamenu built in but you can definitely build one as needed.  I always use Typhoon as my base theme for my client projects and many times i've added a mega menu per the needs of the project.  However, a megamenu is usually very specific to the project, and the struture of the content, so it's not really easy to create a generic one in Typhoon.
